[2024-05-23T15:28:05.601+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-23T15:28:05.613+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-23T15:28:03.754429+00:00 [queued]>
[2024-05-23T15:28:05.617+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-23T15:28:03.754429+00:00 [queued]>
[2024-05-23T15:28:05.617+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-23T15:28:05.625+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): create_db> on 2024-05-23 15:28:03.754429+00:00
[2024-05-23T15:28:05.632+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=85) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-23T15:28:05.632+0000] {standard_task_runner.py:63} INFO - Started process 89 to run task
[2024-05-23T15:28:05.633+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'games', 'create_db', 'manual__2024-05-23T15:28:03.754429+00:00', '--job-id', '694', '--raw', '--subdir', 'DAGS_FOLDER/get_main_games.py', '--cfg-path', '/tmp/tmpsnswzor5']
[2024-05-23T15:28:05.635+0000] {standard_task_runner.py:91} INFO - Job 694: Subtask create_db
[2024-05-23T15:28:05.659+0000] {task_command.py:426} INFO - Running <TaskInstance: games.create_db manual__2024-05-23T15:28:03.754429+00:00 [running]> on host c80b0e71a0a8
[2024-05-23T15:28:05.713+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='games' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-23T15:28:03.754429+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-23T15:28:03.754429+00:00'
[2024-05-23T15:28:05.713+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-23T15:28:05.732+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2024-05-23T15:28:07.470+0000] {logging_mixin.py:188} INFO - Таблица 'games' удалена.
[2024-05-23T15:28:07.898+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-23T15:28:07.899+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-23T15:28:07.905+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=games, task_id=create_db, execution_date=20240523T152803, start_date=20240523T152805, end_date=20240523T152807
[2024-05-23T15:28:07.929+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-23T15:28:07.947+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-23T15:28:07.948+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
